program: |

    fn main() : int {
      return void_func();
    }

    fn void_func() {

    }

expect_compile_failure: true
expect_stderr: |
  3:10 Type error: Return statement has type 'Void' when 'Integer' was expected
    return void_func();
           ^^^^^^^^^
  2:4 Note: Function 'main', declared here, is expected to return 'Integer'
  fn main() : int {
     ^^^^
  Terminating compilation due to 1 error